### PR TITLE
Fix #320

### LINKED ISSUE #320: White textures
### Your GTNH Discord Username

mts2200

### Mod Version

alpha29

### Java Version

Java 21

### Graphics Card Vendor

NVidia

### Bug Report

I upgraded from alpha24 to alpha29 and encountered the following problem:
image
image

My graphics settings:
image
image

### Mod List or GTNH Pack Version

I don't know what mods this exactly works with, but in the screenshot you can see a problem with Thaumcraft, Gadomancy, EvilCraft. This is also can be reproduced with some private mods.

### Final Checklist

- [X] I have searched the issues and haven't found a similar issue.
- [X] I have read the known incompatibilities and this is not related to one of those.
- [X] I am running an officially released version.  (Or, if I've compiled it myself I plan to fix the issue)
- [X] This issue is not related to a feature that is disabed by default - Shaders, MCPF, etc.  [They'll be enabled when they're ready for testing]

### PR DESCRIPTION
**Pull request by mts2200**
This commit fixes the issue. I was not able to figure out the reason why this line was added, but without it, everything works fine.

### CODE REVIEW

**Comment by mitchej123** [comment]
Looking at the commit... it was an attempt to workaround an apparent AMD driver bug... I don't have an AMD card handy to test on so will have to coordinate with @Caedis or someone else.  I"ll see if the tests work without it, and if not, maybe make it AMD specific for the specific CAP that's the problem.

**Comment by Caedis** [comment]
Just ping me on discord when you want me to test

File: src/main/java/com/gtnewhorizons/angelica/glsm/stacks/BooleanStateStack.java [hunk 1]
```diff
@@ -32,7 +32,6 @@ public BooleanStateStack pop() {
         }
 
         set(stack[--pointer]);
-        this.setEnabled(this.enabled);
```
Comment (mitchej123) [inline src/main/java/com/gtnewhorizons/angelica/glsm/stacks/BooleanStateStack.java:35]: Tests on AMD fail without this -- we'll need to make this more targeted to 1) AMD 2) GL_BLEND

Comment (mitchej123) [reply src/main/java/com/gtnewhorizons/angelica/glsm/stacks/BooleanStateStack.java:35]: But on a related note - thanks for tracking down the bug and the fix

**Review by Caedis — APPROVED** [review]
All tests past on AMD now

**Review by mitchej123 — APPROVED** [review]

**Pull request merged**